src/Test.FunctionalTests.Sarif/v2/ConverterTestData/FortifyFpr/ScanWithFailureLevelMatrices.fvdl

```diff
@@ -43576,8 +43576,8 @@ namespace Microsoft.CodeAnalysis.Sarif.Driver
 
         private string RunConverter(StringBuilder sb, string toolFormat, string inputFileName)
         {
-            string expectedFileName = inputFileName + ".sarif";
-            string generatedFileName = inputFileName + ".actual.sarif";
+            string expectedFileName = inputFileName + SarifConstants.SarifFileExtension;
```

Comment:
>SarifConstants.SarifFileExtension; [](start = 54, length = 34)

i am probably splitting hair, but this irks me! The fvdl files added here extracted from the corresponding fpr file (With the same name). These fvdl files are here for reference only (as far as i recall). I think the functional test uses the fpr file as an input. So, let us not change this file so that it is an exact match with what is extracted out of the FPR.

Alternately, we should re-generate the FPR file as well.

Reply:
aah, my memory did not betray me (this time!), see this comment:

[https://github.com/microsoft/sarif-sdk/blob/ffff4b3c9265a441fb00043ebb8e079f6690f870/src/Sarif.Converters/FortifyFprConverter.cs#L127](https://github.com/microsoft/sarif-sdk/blob/ffff4b3c9265a441fb00043ebb8e079f6690f870/src/Sarif.Converters/FortifyFprConverter.cs#L127)

---
In reply to: [406390399](https://github.com/microsoft/sarif-sdk/pull/1850#discussion_r406390399) [](ancestors = 406390399)

Reply:
Oh, good catch! I saw that this code was grayed out and I thought it was an #if block. I will revert this change.

---
In reply to: [406391355](https://github.com/microsoft/sarif-sdk/pull/1850#discussion_r406391355) [](ancestors = 406391355,406390399)